app/src/main/java/popularmovies/one/fabianreddig/udacity/projectone/movieactivity/MovieActivity.java

```diff
@@ -0,0 +1,31 @@
+package popularmovies.one.fabianreddig.udacity.projectone.movieactivity;
+
+import android.os.Bundle;
+import android.support.v4.app.Fragment;
+import android.support.v4.app.FragmentTransaction;
+import android.support.v4.content.LocalBroadcastManager;
+import android.support.v7.app.AppCompatActivity;
+
+import javax.inject.Inject;
+
+import popularmovies.one.fabianreddig.udacity.projectone.PopularMoviesApplication;
+import popularmovies.one.fabianreddig.udacity.projectone.R;
+
+public class MovieActivity extends AppCompatActivity {
+
+    @Inject
+    LocalBroadcastManager broadcastManager;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_movie);
+        PopularMoviesApplication.applicationComponent().inject(this);
+        switchFragment(MovieListFragment.newInstance());
```

Comment:
Should only need to commit a fragment when savedInstanceState == null right? the fragment manager will auto-recreate its fragments on config change